### ISSUE TITLE
Sort by column not working except for "name"

### ISSUE THREAD

**Issue by matt32106** [GoldenGnu/jeveassets#40]
when i click on the top of any column except "name", it does not sort, is it a known issue? can somebody confirm this?

jeveassets 5.4.2 (2018-01-10)
ubuntu 16.04 LTS
4.4.0-112-generic
openjdk-8-jdk

**Comment by GoldenGnu** [comment]
Take a look at the FAQ: https://github.com/GoldenGnu/jeveassets/wiki/ReadMe#sort-columns
Let me know if that solves your problem

**Comment by matt32106** [comment]
Yes, it solves the problem.
However, the current logic is not very intuitive, it is the first app i meet that does that (and i have seen my share in the last 30 years :) ). May i suggest the following?
- maybe you could consider reversing the logic (single click sorts, double click sub-sorts?)
- if you don't want to implement this change, at least, there should be some kind of tooltip poping when the mouse hovers over the column names.

Thanks for great app btw!

**Comment by GoldenGnu** [comment]
I agree with it being unintuitive and it have been in the FAQ for years for the same reason.

I will look into adding tool tips and/or changing the click logic, but, I think the click logic is outside my control as I use a library for sorting and filtering the tables.

Thank you for your feedback

**Comment by GoldenGnu** [comment]
Looks like you can make your own SortingStrategy:
https://github.com/glazedlists/glazedlists/tree/master/core/src/main/java/ca/odell/glazedlists/impl/gui
